src/services/services.ts

```diff
@@ -6740,7 +6741,7 @@ namespace ts {
                 fileName: node.getSourceFile().fileName,
                 textSpan: createTextSpanFromBounds(start, end),
                 isWriteAccess: isWriteAccess(node),
-                isDefinition: isDeclarationName(node)
+                isDefinition: isDeclarationName(node) || node.parent.kind === SyntaxKind.ComputedPropertyName

```

Comment:
I tried pushing it further, but computed property's symbols started disappearing from our symbol baselines. I did discover `isLiteralCompuatedPropertyDeclarationName`, so I switched to that instead of the direct check of SyntaxKind.